source/manual/mysql.html.md

```diff
@@ -5,49 +5,55 @@ section: Backups
-"hot" backups and uses the [xbstream](https://www.percona.com/doc/percona-xtrabackup/2.3/xbstream/xbstream.html) tool to stream data to STDOUT. We redirect this output
-into a file stored in an Amazon S3 bucket using a tool written in Go called [gof3r](https://github.com/rlmcpherson/s3gof3r). Xtrabackup has an encryption
-function that we can use to encrypt the backups by providing an encryption key, and we also ensure we have serverside encryption in the S3 bucket. The way that
-the backups are piped straight to S3 means that they never touch the disk so we do not have to worry about stuff like disk usage.
+We use a tool called [Innobackupex](https://www.percona.com/doc/percona-xtrabackup/2.2/innobackupex/incremental_backups_innobackupex.html) which is a wrapper for [Xtrabackup](https://www.percona.com/doc/percona-xtrabackup/2.3/index.html). 
 
-This method was inspired by this [blog post from MariaDB](https://mariadb.com/blog/streaming-mariadb-backups-cloud).
+### Backing up
+Innobackupex takes binary "hot" backups and uses the [xbstream](https://www.percona.com/doc/percona-xtrabackup/2.3/xbstream/xbstream.html) tool to stream data to STDOUT. We redirect this output into a file stored in an Amazon S3 bucket using a tool written in Go called [gof3r](https://github.com/rlmcpherson/s3gof3r). 
 
-We use the concept of [incremental backups which are built in the toolset](https://www.percona.com/doc/percona-xtrabackup/2.2/xtrabackup_bin/incremental_backups.html).
-Each night we take a "base" backup, and then every n time after that (default: 15 minutes) we take an "incremental" backup. To restore the backup we would be able to get
-the "base" backup, and then apply any number of "incremental" backups on top of it.
+Each night we take a "base" backup, and then every _n_ time after that (default: 15 minutes) we take an "incremental" backup. 
 
-The drawback of this method is that restores are more complicated.
+### Restoring
 
-To make this easier, a [script has been written](https://github.com/alphagov/govuk-puppet/blob/master/modules/govuk_mysql/templates/usr/local/bin/xtrabackup_s3_restore.erb) which will automatically get the very latest base backup, prepare it, and then fetch the latest incremental backup, prepare there and then copy them both back to the MySQL
-data directory.
+To restore the backup we use a [script](https://github.com/alphagov/govuk-puppet/blob/master/modules/govuk_mysql/templates/usr/local/bin/xtrabackup_s3_restore.erb) to retrieve the base backup and then apply any number of incremental backups on top of it. 
 
-An advantage of this method is that because it is a binary type of backup, restores are much quicker than having to import SQL text based backups.
+Under the hood the script completes the following steps:
+
+- Retrieve the latest base backup.
+- Fetch the latest incremental backups.
+- Copy the consolidated backup (base plus incremental) to the MySQL data directory.
+
+
+### Why this approach
```

Comment:
Thank you for adding the headings in - so much more scannable!

Reply:
😁